src/Components/SignatureSideBar/Components/ClassApplyBtn.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components";
+import { Button, ButtonColor } from '@class101/ui';
+
+function ClassApplyBtn() {
+  return (
+    <ApplyBtnContainer>
+      <Button color={ButtonColor.RED}><span>클래스 신청하기</span></Button>
```

Comment:
```suggestion
      <Button color={ButtonColor.RED}>
        <span>클래스 신청하기</span>
      </Button>
```
부모 자식 관계가 잘 드러날 수 있도록 줄바꿈 해주세요